### PR TITLE
March 22 Fix

### PR DESCRIPTION
**Pull request by actionschnitzel**
- Folder structure improved
- /home/pi/ is replaced with $HOME
- APT Fast Install has now a search funktion + combobox
- Lot of code fixes
- Icon & PNG improvements
- Shop currently disabled due to makeover(I'm working on it)
- More infos on the help button
- New Overclocking option
- Overclocking Berrys are now 
  locked until you reset it / Close Pigro / Or reboot
- NEW OVERCLOCKING OPTION:
  arm_freq=2300
  gpu_freq=500
  over_voltage=14
  force_turbo=1

### CODE REVIEW

File: splash.py [hunk 1]
```diff
@@ -11,43 +11,57 @@
...
-    pip_install('distro')
-    pip_install('psutil')
-    #pip_install('pywebview')
-    isInstalling=False
+    c.installIfNotInstalled("xterm")
+    c.installIfNotInstalled("python3-pil")
+    c.installIfNotInstalled("python3-pil.imagetk")
+    c.installIfNotInstalled("python3-pip")
+    c.installIfNotInstalled("mpg123")
+    c.installIfNotInstalled("lolcat")
```
Comment (Itai-Nelken) [inline splash.py:58]: Could be replaced with
```python
for pkg in ["xterm", "python3-pil", "python3-pil.imagetk", "python3-pip", "mpg123", "lolcat"]:
    c.installIfNotInstalled(pkg)
```

**Review by Itai-Nelken — COMMENTED** [review]
A bit late, so for the next update.

File: splash.py (hunk 1 above)
Comment (actionschnitzel) [reply splash.py:58]: I'll add it to the next update. This update was really exhausting to write. I need a little break first xD

Comment (Itai-Nelken) [reply splash.py:58]: You definitely deserve a break!

**Pull request merged**